peps/pep-0775.rst

```diff
@@ -0,0 +1,136 @@
+PEP: 775
+Title: Make zlib required to build Python
+Author: Stan Ulbrych <stanulbrych@gmail.com>,
+        Petr Viktorin <encukou@gmail.com>
+Discussions-To: https://discuss.python.org/t/lets-make-zlib-required-rather-than-optional-to-build-cpython/23062
+Status: Draft
+Type: Standards Track
+Created: 20-Feb-2025
+Python-Version: 3.14
+Post-History: `23-Jan-2023 <https://discuss.python.org/t/23062>`__
+
+Abstract
+========
+
+Building CPython on systems without the `zlib <https://zlib.net>`_ compression library, except WASI,
+is no longer supported.
+
+The ``zlib`` module is made a required part of the standard library,
+except on WASI.
+
+Builds without ``zlib`` will still be possible, but formally unsupported.
+
+
+Motivation
+==========
+
+The ``zlib`` library, which powers the ``zlib`` Python module,
```

Comment:
The zlib compression library doesn't need code formatting (re: https://zlib.net, https://en.wikipedia.org/wiki/Zlib)

```suggestion
The zlib library, which powers the :mod:`!zlib` Python module,
```